reading of a file using fgets() .c

```diff
@@ -0,0 +1,17 @@
+#include<stdio.h>
+main()
+{
+  char str[10];
+  file *fp;
+  fp = fopen("f1.txt"."r");
```

Comment:
Kindly type statement 5 and 6 in one statement as "FILE *fp = fopen("f1.txt","r")"